### PR TITLE
Documentationv2.1

### PR DESCRIPTION
**Pull request by AnuragSahu11**
*Added dark mode to the CL
*Added JS for the hamburger icon for responsiveness

### CODE REVIEW

File: documentation/documentation.html [hunk 1]
```diff
@@ -6,34 +6,40 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="/main.css" />
     <link rel="stylesheet" href="style.css" />
+
     <title>Lucid Ui | Documentation</title>
   </head>
-  <body>
+  <body class="">
```
Comment (shivampandey0) [inline documentation/documentation.html:12]: empty class

File: documentation/documentation.html [hunk 2]
```diff
@@ -6,34 +6,40 @@
...
     <!-- Navbar -->
     <nav class="navbar li-shadow">
       <div class="nav-brand">
         <a href="" class="logo-a">
-          <i class="fas fa-suitcase"></i>
+          <img
+            class="logo-s"
+            src="/documentation/02 - Colorful - Icon Only.png"
+            alt=""
```
Comment (shivampandey0) [inline documentation/documentation.html:20]: empty `alt` attribute

**Review by shivampandey0 — COMMENTED** [review]
Dark mode looks good.
Added some suggestions.

File: documentation/documentation.html (hunk 1 above)
Comment (AnuragSahu11) [reply documentation/documentation.html:12]: yes I will change that

File: documentation/documentation.html (hunk 2 above)
Comment (AnuragSahu11) [reply documentation/documentation.html:20]: I will add to it, thanks

**Comment by rishichourasia** [comment]
looks very good to me so far , great features 👍

**Review by rishichourasia — APPROVED** [review]

**Pull request merged**